src/app/beer_garden/api/stomp/transport.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import stomp
+from brewtils.models import Operation
+from brewtils.schema_parser import SchemaParser
+from typing import Tuple, Any
+
+import beer_garden.events
+import beer_garden.router
+
+logger = logging.getLogger(__name__)
+
+
+def consolidate_headers(*args) -> dict:
+    """Consolidates header dictionaries into one dict
+
+    Args:
+        The iterable of header dictionaries to combine
+
+    Returns:
+
+    """
+    headers = {}
+
+    for header_dict in args:
+        if header_dict:
+            headers.update(header_dict)
+
+    return headers
+
+
+def process(body) -> Tuple[str, dict]:
+    """Processes a message body prior to sending
+
+    We always want to send Operations. So if the given message is an Event we'll wrap
+    it in an Operation.
+
+    Args:
+        body: the message body to process
+
+    Returns:
+        Tuple of the serialized message and response headers dict
+
+    """
+    many = isinstance(body, list)
+
+    if body.__class__.__name__ == "Event":
+        body = Operation(operation_type="PUBLISH_EVENT", model=body, model_type="Event")
+
+    model_class = (body[0] if many else body).__class__.__name__
+
+    if not isinstance(body, str):
+        body = SchemaParser.serialize(body, to_string=True, many=many)
+
+    return body, {"model_class": model_class, "many": many}
+
+
+def send(
+    body: Any,
+    garden_headers: dict = None,
+    conn: stomp.Connection = None,
+    send_destination: str = None,
+    request_headers: dict = None,
+):
+    message, model_headers = process(body)
+
+    headers = consolidate_headers(model_headers, request_headers, garden_headers)
```

Comment:
Should change order so the result response sends the correct `model_class` in the header, since a child garden will send a request object as a response from receiving a request create operation.